src/applications/fry-dea/helpers.jsx

```diff
@@ -85,3 +86,46 @@ export const applicantIsChildOfSponsor = formData => {
 
   return sponsor?.relationship === RELATIONSHIP.CHILD;
 };
+
+export const AdditionalConsiderationTemplate = (
+  title,
+  path,
+  formField,
+  UI,
+  SCHEMA,
+  dependA,
+  dependB = false,
+  condition2 = false,
+) => {
+  return {
+    title,
+    path,
+    uiSchema: {
+      'view:subHeadings': {
+        'ui:description': <h3>{title}</h3>,
+      },
+      [formField]: {
+        ...UI,
+      },
+    },
+    schema: {
+      type: 'object',
+      required: [formField],
+      properties: {
+        'view:subHeadings': {
+          type: 'object',
+          properties: {},
+        },
+        [formField]: { ...SCHEMA },
+      },
+    },
+    depends: formData => {
```

Comment:
I think it's good to abstract some of these redundant form configurations but this code isn't terribly readable.